### PR TITLE
Dailymotion Bid Adaptor: initial release

### PR DESCRIPTION
**Pull request by kvnsw**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter : docs PR (prebid.github.io) https://github.com/dailymotion-oss/prebid.github.io/pull/2
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

A new bidder adapter dailymotionBidAdapter added. 

- contact email of the adapter’s maintainer : **ad-leo-engineering@dailymotion.com**

- Test parameters for validating bids:

By setting the following configuration options, you'll get a constant response to any request to validate your adapter integration:

```
config.setConfig({
  dailymotion: {
    api_key: 'dailymotion-testing',
    position: 'preroll'
  }
});
```

```
const adUnits = [{
  code: 'midroll',
  mediaTypes: {
    video: {
      context: 'instream',
      playerSize: [1280, 720],
    },
  },
  bids: [{
    bidder: "dailymotion",
    params: {}
  }]
}];
```

Please note that failing to set these configuration options will result in the adapter not bidding at all.

### CODE REVIEW

**Review by Aditi0101 — CHANGES_REQUESTED** [review]
Please do not merge : We are still waiting for email and prebid service changes.

File: modules/dailymotionBidAdapter.js
Comment (jeremy-le-dev) [inline modules/dailymotionBidAdapter.js]: We could have something like that `buildRequests: (validBidRequests = [], bidderRequest) => validBidRequests.map(bid => ({` but as you did it works 👍🏻

**Review by jeremy-le-dev — APPROVED** [review]

File: modules/dailymotionBidAdapter.md
Comment (kvnsw) [inline modules/dailymotionBidAdapter.md]: Same as https://github.com/dailymotion-oss/prebid.github.io/pull/1/files#r1457209383

File: modules/dailymotionBidAdapter.md
Comment (kvnsw) [inline modules/dailymotionBidAdapter.md]: Same as https://github.com/dailymotion-oss/prebid.github.io/pull/1/commits/db85372c781719c528be270cef961fd03c48424d

File: modules/dailymotionBidAdapter.md
Comment (Aditi0101) [inline modules/dailymotionBidAdapter.md:6]: We need to update email here as well.

File: modules/dailymotionBidAdapter.md
Comment (jeremy-le-dev) [inline modules/dailymotionBidAdapter.md:4]: "Dailymotion **Bid** Adapter" or "Dailymotion **Prebid** Adapter" ? 
In the description we have "prebid"

Comment (Aditi0101) [reply modules/dailymotionBidAdapter.md:4]: Dailymotion Bid Adapter is ok. Other bid adapter also has similar name.

File: modules/dailymotionBidAdapter.md
Comment (Aditi0101) [reply modules/dailymotionBidAdapter.md:6]: We want to keep Maintainer: ad-leo-engineering@dailymotion.com. so LGTM.

**Review by Aditi0101 — APPROVED** [review]

**Review by jeremy-le-dev — APPROVED** [review]

**Review by sebmil-daily — APPROVED** [review]

**Pull request merged**
